src/pdf.rs

```diff
@@ -50,11 +50,11 @@ impl PdfSurface {
 
     pub fn get_versions() -> impl Iterator<Item = PdfVersion> {
         let vers_slice = unsafe {
-            let mut vers_ptr: *mut ffi::cairo_pdf_version_t = mem::uninitialized();
+            let mut vers_ptr = mem::MaybeUninit::<*mut ffi::cairo_pdf_version_t>::uninit();
             let mut num_vers = 0;
```

Comment:
This one can be `MaybeUninit::uninit()` for consistency with how we do things elsewhere.